socorro/external/crashstorage_base.py

```diff
@@ -589,13 +590,27 @@ def save_processed(self, processed_crash):
     #--------------------------------------------------------------------------
     def save_raw_and_processed(self, raw_crash, dump, processed_crash,
                                crash_id):
+        storage_exception = PolyStorageError()
         for a_store in self.stores.itervalues():
-            a_store.save_raw_and_processed(
-              raw_crash,
-              dump,
-              processed_crash,
-              crash_id
-            )
+            self.logger.debug('writing to %s' % (
+                hasattr(a_store, 'wrapped_object') and
```

Comment:
If it's been helpful, you can elevate it to `info` instead. 

I think we're abusing the logging system. A better pattern, in general, is to have lots of `logging.debug(...)` and stuff in the code and some config switch you can flip on to enable it all and when you're done, you flip it back to just `info`. 

So, what I'm saying is, perhaps leave that line as is (it's not a performance penalty if logging is configured to just drop it) and some day better tackle the ability to, easily, switch from regular to debug mode across our servers.